test/common/flags.test.ts

```diff
@@ -134,4 +134,24 @@ describe('requiredDoceOrgFlag', () => {
       expect(err.message).to.include(`No authorization information found for ${invalidAlias}`);
     }
   });
+
+  it('fails when no alias is provided and the --target-devops-center config var is set but invalid', async () => {
+    const invalidTargetDevopsCenter = 'invalid-target-devops-center';
+    sandbox.stub(ConfigAggregator.prototype, 'getInfo').returns({
+      value: 'invalid-target-devops-center',
+      key: ConfigVars.TARGET_DEVOPS_CENTER,
+      isLocal: () => false,
+      isGlobal: () => true,
+      isEnvVar: () => false,
+    });
+    try {
+      await Parser.parse(['--requiredDoceOrg='], {
```

Comment:
I cannot find this flag. What is the difference from devops-center-username?

Reply:
Here I'm unit testing the callback function for the parser, it is not tied to any particular flag. So I opted to use a flag name that indicates what the callback does instead of using the flag name from our command.

Reply:
Ok!